src/reactor/fns.js

```diff
@@ -168,6 +169,8 @@ export function loadState(reactorState, state) {
 export function addObserver(observerState, getter, handler) {
   // use the passed in getter as the key so we can rely on a byreference call for unobserve
   const getterKey = getter
+
+  getter = convertToGetterLiteral(getter)
```

Comment:
I really dislike how we have to use `convertToGetterLiteral` everywhere.  It's also not a free operation since we do assertion checking that it's infact a valid getter.